shell/platform/embedder/embedder.cc

```diff
@@ -798,6 +798,8 @@ FlutterEngineResult FlutterEngineRun(size_t version,
               int64_t texture_identifier, GrContext* context,
               const SkISize& size) -> sk_sp<SkImage> {
         FlutterOpenGLTexture texture = {};
+        texture.width = 0;
```

Comment:
Nit: You don't need to zero it out again. The `= {}` call above has done this for you already.